README.md

```diff
@@ -35,18 +35,18 @@ As a strawman, we propose **the 99th percentile (P99)** as a lowest common denom
+As explained in the introduction, defining a delay metric is not just about choosing a percentile. The layer to measure and the background traffic pattern to use also have to be defined. As soon as these have been settled on, researchers, product engineers, etc. tend to optimize around this set of conditions---the so-called 'benchmark effect'. It is possible that harmonizing around one choice of percentile will lead to a benchmark effect. However, a percentile metric seems robust against such perverse incentives, because it seems hard to contrive performance results that fall off a cliff just beyond a certain percentile. [This is a little weak.  An intermittent outtage on a link that causes packets to queue up would cause delay spikes that might only show up in percentiles greater than P99.  These outtages would be ignored if P99 were the metric.] Nonetheless, even if there were a benchmark effect, it would be harmless if the percentile chosen for the benchmark realistically reflected the needs of most applications. {ToDo: better wording for last sentence.}
 
 ## How to articulate a percentile to the public?
 
 Delay is not an easy metric for public consumption, because it exhibits the following undesirable features:
 
 * Larger is not better.
-  * It might be possible to invert the metric [[RPM21](#RPM21)], but rounds per minute carries an implication that it is only for repetitive tasks, which would limit the scope of the metric
+  * It might be possible to invert the metric [[RPM21](#RPM21)], but rounds per minute carries an implication that it is only for repetitive tasks for which average delay is important, which would limit the scope of the metric 
 * it is measured in time units (ms) that seem too small to matter, and which are not common currency for a lay person
   * This might also be addressed by inverting the metric
 
-A delay percentile is expressed as a delay, so it shares the same failings, and the same potential mitigations. But a percentile carries additional baggage as well:
+A delay percentile is expressed as a delay, so it shares the same failings, and the same potential mitigation [huh? it's hard for me to see how one would invert a P99 metric in a useful way]. But a percentile carries additional baggage as well:
```

Comment:
I forgot to mention this in my email, but I agree that inverting the metric is probably not the direction.
Delay in milliseconds is perfectly fine I think and I think users know for latency, lower is better. Inverting it could make it more confusing.

Reply:
That is a great comment!   I agree.  Anyone with a 3rd grade education understands that for some metrics (like cost, for example), lower is better. Let's strike any concern about latency being a 'smaller is better' metric.  The other concern (units are really small) is a true concern and could be addressed more directly.

Reply:
I'll certainly remove the stuff about inverting the metric, and I'll merge the discussion about "higher is better" into the bullet about the units seeming too small to matter.